README.md

```diff
@@ -1,20 +1,26 @@
-# echidna
+# Echidna: A Fast Smart Contract Fuzzer <a href="https://raw.githubusercontent.com/crytic/echidna/master/echidna.png"><img src="https://raw.githubusercontent.com/crytic/echidna/master/echidna.png" width="75"/></a>
 
 ![Build Status](https://github.com/crytic/echidna/workflows/CI/badge.svg)
-![echidna logo](echidna.png)
 
 Echidna is a weird creature that eats bugs and is highly electrosensitive (with apologies to Jacob Stanley)
 
-More seriously, Echidna is a Haskell library designed for fuzzing/property-based testing of EVM code. It supports relatively sophisticated grammar-based fuzzing campaigns to falsify a variety of predicates.
+More seriously, Echidna is a Haskell program designed for fuzzing/property-based testing of Ethereum smarts contracts. It uses a relatively sophisticated grammar-based fuzzing campaigns based on a contract ABI to falsify Solidity assertions or user-defined predicates. We designed Echidna with modularity in mind, so it can be easily extended to include new mutations or test specific contracts in specific cases.
```

Comment:
"We designed Echidna with modularity in mind, so it can be easily extended to include new mutations or test specific contracts in specific cases." By extend you mean in the Haskell code or some user facing feature?

Reply:
Maybe we can put some educational links for newcomers? For example this https://solidity.readthedocs.io/en/develop/abi-spec.html for "contract ABI"

Reply:
> "We designed Echidna with modularity in mind, so it can be easily extended to include new mutations or test specific contracts in specific cases." By extend you mean in the Haskell code or some user facing feature?

I was referring to Haskell code. This sentence was added to address @incertia comment (so feel free to suggest something better :smile: ).

> Maybe we can put some educational links for newcomers? For example this https://solidity.readthedocs.io/en/develop/abi-spec.html for "contract ABI"

Sure.

Reply:
Done.

Reply:
I would just include this is regarding Haskell code, some might get an impression that there are extra user facing features